connectors/mediasession/src/main/java/com/theoplayer/android/connector/mediasession/MediaMetadataProvider.kt

```diff
@@ -1,44 +1,47 @@
 package com.theoplayer.android.connector.mediasession
 
+import android.graphics.Bitmap
 import android.support.v4.media.MediaMetadataCompat
 import android.util.Log
 import com.theoplayer.android.api.source.SourceDescription
-import com.theoplayer.android.api.source.metadata.MetadataDescription
-
-val PROP_METADATA = hashMapOf(
-    MediaMetadataCompat.METADATA_KEY_ADVERTISEMENT to "advertisement",
-    MediaMetadataCompat.METADATA_KEY_ALBUM to "album",
-    MediaMetadataCompat.METADATA_KEY_ALBUM_ARTIST to "albumArtist",
-    MediaMetadataCompat.METADATA_KEY_ALBUM_ART_URI to "artUri",
-    MediaMetadataCompat.METADATA_KEY_AUTHOR to "author",
-    MediaMetadataCompat.METADATA_KEY_ARTIST to "artist",
-    MediaMetadataCompat.METADATA_KEY_COMPILATION to "compilation",
-    MediaMetadataCompat.METADATA_KEY_COMPOSER to "composer",
-    MediaMetadataCompat.METADATA_KEY_DATE to "date",
-    MediaMetadataCompat.METADATA_KEY_DISC_NUMBER to "discNumber",
-    MediaMetadataCompat.METADATA_KEY_DISPLAY_DESCRIPTION to "displayDescription",
-    MediaMetadataCompat.METADATA_KEY_DISPLAY_ICON_URI to "displayIconUri",
-    MediaMetadataCompat.METADATA_KEY_DISPLAY_SUBTITLE to "displaySubtitle",
-    MediaMetadataCompat.METADATA_KEY_DISPLAY_TITLE to "displayTitle",
-    MediaMetadataCompat.METADATA_KEY_DOWNLOAD_STATUS to "downloadStatus",
-    MediaMetadataCompat.METADATA_KEY_DURATION to "duration",
-    MediaMetadataCompat.METADATA_KEY_GENRE to "genre",
-    MediaMetadataCompat.METADATA_KEY_MEDIA_ID to "mediaId",
-    MediaMetadataCompat.METADATA_KEY_MEDIA_URI to "mediaUri",
-    MediaMetadataCompat.METADATA_KEY_NUM_TRACKS to "numTracks",
-    MediaMetadataCompat.METADATA_KEY_TITLE to "title",
-    MediaMetadataCompat.METADATA_KEY_TRACK_NUMBER to "trackNumber",
-    MediaMetadataCompat.METADATA_KEY_WRITER to "writer",
-    MediaMetadataCompat.METADATA_KEY_YEAR to "year",
-)
 
+private const val PROP_ALBUM = "album"
```

Comment:
Why advertisement, artUri, duration properties are missing?

I see that we set advertisement and duration from the player state, but maybe we should copy them from metadata also? I didn't find where we copy artUri btw.